bitnami/redis/templates/configmap-scripts.yaml

```
@@ -294,28 +294,79 @@ data:
     #!/bin/bash
 
     . /opt/bitnami/scripts/libvalidations.sh
```

```suggestion
    . /opt/bitnami/scripts/libvalidations.sh
    . /opt/bitnami/scripts/libos.sh
```

This will be required for the comment about using `retry_while`